docs/management/managing-remote-clusters.asciidoc

```diff
@@ -1,26 +1,64 @@
 
-{kib} *Management* provides two user interfaces for working with data from remote 
-clusters. 
+* {xpack-ref}/xpack-ccr.html[Cross cluster replication]
+* {ref}/modules-cross-cluster-search.html[Cross cluster search]
+* {xpack-ref}/cross-cluster-configuring.html[Cross cluster security requirements]
+
+[float]
+[[managing-remote-clusters]]
+== Managing remote clusters 
 
 *Remote Clusters* helps you manage remote clusters for use with 
-{ref}/modules-cross-cluster-search.html[cross cluster search] and 
-{xpack-ref}/xpack-ccr.html[cross cluster replication]. You can add and remove remote 
+cross cluster search and cross cluster replication. You can add and remove remote 
 clusters and check their connectivity. 
 
+Before you use this feature, you should be familiar with the concept of 
+{ref}/modules-remote-clusters.html[remote clusters].  
+ 
 Go to *Management > Elasticsearch > Remote Clusters* to get started. 
 
 [role="screenshot"]
 image::images/add_remote_cluster.png[][UI for adding a remote cluster]
 
+Once a remote cluster is registered, you can use the tools under *Cross Cluster Replication* 
+to add and manage follower indices on the local cluster, and replicate data from 
+indices on the remote cluster based on an auto-follow index pattern.
+
+[float]
+[[managing-cross-cluster-replication]]
+== Managing cross cluster replication
+ 
+*Cross Cluster Replication* includes tools to help you create and manage the cross cluster 
+replication process. If you want to replicate data from existing indices, or set up 
+local followers on a case-by-case basis, you can create and manage followers individually 
+from *Follower indices*. If you want to automatically detect and follow new indices 
+when they are created on a remote cluster, you can do so from *Auto-follow patterns*. 
 
-*Cross Cluster Replication* includes tools to help you create and manage the remote 
-replication process.  You can follow an index pattern on the remote cluster for 
-auto-discovery and then replicate new indices in the local cluster that match the 
-auto-follow pattern.
+Creating an auto-follow pattern can be useful when you have time-series data, like a logs index, on the 
+remote cluster that gets created or rolled over on a daily basis. Once you have configured an 
+auto-follow pattern, any time a new index with a name that matches the pattern is 
+created in the remote cluster, a follower index is automatically configured in the local cluster.
+
+From the same view, you can also see a list of your saved auto-follow patterns for 
+a given remote cluster, and monitor to see if the replication is active.
+
+Before you use these features, you should be familiar with the following concepts:
+
+* {xpack-ref}/ccr-requirements.html[Requirements for leader indices] 
+* {xpack-ref}/ccr-auto-follow.html[Automatically following indices]
 
 Go to *Management > Elasticsearch > Cross Cluster Replication* to get started. 
```

Comment:
```suggestion
To get started, go to *Management > Elasticsearch > Cross Cluster Replication*. 
```

Reply:
This is different than the pattern we've been using in other doc: https://www.elastic.co/guide/en/kibana/current/creating-index-lifecycle-policies.html